### PR TITLE
chore(types): webdriver typings for locators

### PR DESCRIPTION
**Pull request by cnishina**
- temporarily add typings for selenium-webdriver.d.ts
- include selenium-webdriver dependency to built/index.d.ts
- add webdriver typings to locators
- update example and spec/install to not use typings.json
  - spec test updated to get the tsc test to pass
- includes clang formatting fixes

### CODE REVIEW

File: lib/element.ts [hunk 1]
```diff
@@ -964,9 +961,9 @@ export class ElementFinder extends WebdriverWebElement {
    * @returns {ElementFinder} which resolves to whether
```
Comment (juliemr) [inline lib/element.ts]: Can you update this comment too?

File: gulpfile.js [hunk 2]
```diff
@@ -88,7 +88,8 @@ gulp.task('types', function(done) {
   var files = ['browser', 'element', 'locators', 'expectedConditions',
     'config', 'plugins', 'ptor'];
   var outputFile = path.resolve(folder, 'index.d.ts');
-  var contents = '';
+  var contents = '/// <reference path="../typings/index.d.ts" />\n';
```
Comment (juliemr) [inline gulpfile.js]: Can you explain why we need this and didn't need it before?

File: typings/index.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+/// <reference path="globals/selenium-webdriver/index.d.ts" />
```
Comment (juliemr) [inline typings/index.d.ts]: Can we add a note that these are forked from definitely typed so we can upkeep them?

File: lib/element.ts (hunk 1 above)
Comment (cnishina) [inline lib/element.ts]: Done. Updated comment for both `isPresent` and `isElementPresent`.

File: gulpfile.js (hunk 2 above)
Comment (cnishina) [inline gulpfile.js]: As discussed, triple slash reference path brings in the ambient typings. We are doing this specifically for selenium-webdriver typings.

File: typings/index.d.ts (hunk 3 above)
Comment (cnishina) [inline typings/index.d.ts]: Added a readme file. We will remove this folder when we can install these typings with `npm install @types/selenium-webdriver@2.53.1`.

**Pull request merged**